src/intrinsics/SetPrototype.mts

```diff
@@ -317,7 +671,7 @@ function* GetSetRecord(obj: Value): PlainEvaluator<SetRecord> {
 /** https://tc39.es/proposal-set-methods/#sec-getkeysiterator */
 function* GetKeysIterator(setRec: SetRecord): PlainEvaluator<IteratorRecord> {
```

Comment:
Hi, I cannot find this AO in the formal spec, would you like to investigate what happened? Thanks!